src/main/java/GoToJailSquare.java

```diff
@@ -0,0 +1,25 @@
+public class GoToJailSquare extends Square {
+    private Square jail;
+
+    /**
+     *
+     * Square constructor
+     *
+     * @param name String : name of the square
+     * @param jail Square : jail square
+     */
+    public GoToJailSquare(String name, Square jail) {
```

Comment:
if we have only one jail square, it can be a private member without passing as parameter

Reply:
The goToJailSquare can't know who the jail square is because he doesn't have access to the board, it's the board that must tell him